src/recommender.py

```diff
@@ -470,10 +479,12 @@ def execute(self, arguments=None, persist=True, check_license=False):
 
             # Call PGM and get the response
             start = datetime.datetime.utcnow()
-            insights_response = self.call_insights_recommender(input_task_for_insights_recommender)
-            elapsed_seconds = (datetime.datetime.utcnow() - start).total_seconds()
-            msg = 'It took {t} seconds to get response from PGM ' \
-                  'for external request {e}.'.format(t=elapsed_seconds,
+            insights_response = self.call_insights_recommender(
+                input_task_for_insights_recommender)
+            elapsed_seconds = (datetime.datetime.utcnow() -
```

Comment:
Do we need this?

Reply:
What are we testing/debugging here?

Reply:
And do we need to test it on statging/prod as well?

Reply:
Yes, it shows up in the logs on stage and prod.

Reply:
@miteshvp @samuzzal-choudhury Might be able to throw more light on it since these statements were added in the beginning.